specification/containerservice/resource-manager/Microsoft.ContainerService/aks/stable/2024-08-01/managedClusters.json

```diff
@@ -1195,6 +1195,9 @@
           },
           {
             "$ref": "#/parameters/AgentPoolNameParameter"
+          },
+          {
+            "$ref": "#/parameters/IgnorePodDisruptionBudgetParameter"
```

Comment:
should this be added to managed cluster delete too?

Reply:
We will not enable this option for MC delete as it is not so necessary.